app/javascript/app/components/emission-pathways-scenario-table/emission-pathways-scenario-table-styles.scss

```diff
@@ -0,0 +1,25 @@
+@import '~styles/layout.scss';
+@import '~styles/settings.scss';
+
+.col4 {
+  @extend %grid;
+
+  @include msGridColumns(1fr, 1fr, 1fr, 1fr);
+
+  grid-template-columns: repeat(4, 1fr);
+  align-items: end;
+}
+
+.loader {
+  min-height: 475px;
+}
+
+.searchBox,
+.uploadButton {
+  grid-column-end: -1;
+}
+
+.noContent {
+  min-height: 460px;
```

Comment:
Also can you match this to the min height of the table please so that you dont see a jump when searching